src/umzug.ts

```diff
@@ -332,23 +333,28 @@ export class Umzug<Ctx extends object = object> extends emittery<UmzugEvents<Ctx
 		});
 	}
 
+	private getDefaultPrefix(prefix: MigrationPrefix) {
+		const isoDate = new Date().toISOString();
+		const prefixes = {
+			TIMESTAMP: isoDate.replace(/\.\d{3}Z$/, '').replace(/\W/g, '.'),
+			DATE: isoDate.split('T')[0].replace(/\W/g, '.'),
+			NONE: '',
+		};
+		return prefixes[prefix];
+	}
+
 	async create(options: {
 		name: string;
 		folder?: string;
-		prefix?: 'TIMESTAMP' | 'DATE' | 'NONE';
+		prefix?: MigrationPrefix;
```

Comment:
It's a bit confusing having two places to setup the prefix. Can we deprecate this one with a warning message?